.github/workflows/test.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     - name: build test wrapper
       run: ./tool/go build -o /tmp/testwrapper ./cmd/testwrapper
     - name: integration tests as root
-      run: PATH=$PWD/tool:$PATH /tmp/testwrapper --sudo ./tstest/integration/ -race
+      run: PATH=$PWD/tool:$PATH /tmp/testwrapper --exec "sudo -E" ./tstest/integration/ -race
```

Comment:
I don't like putting the package path(s) in the middle of the flags. Move `-race` next to `-exec` and use consistent hyphen count between the two/everywhere.

Reply:
done